LICENSE

```diff
@@ -176,6 +176,7 @@
 
    END OF TERMS AND CONDITIONS
 
+   Copyright 2019, 2020 The Linux Foundation
```

Comment:
Looking at the list of [recommended options](https://github.com/cncf/foundation/blob/master/copyright-notices.md#copyright-notices)

Perhaps "Copyright 2019, 2020 The go-digest contributors" ?

(I personally like "authors" or "contributors", as it more clearly describes that copyright is not transferred to `<some organisation>`, but remains with the author/contributor)